cmake/ios/ios_test.sh.in

```diff
@@ -121,9 +121,10 @@ while getopts "a:b:c:p:h" opt; do
     a)
         ARGUMENTS=$OPTARG
         ;;
-    b)
-        BUILD_FLAG=$OPTARG
-        ;;
+    ## Temporarily disabled until iOS build jobs will be configured
+    # b)
+    #     BUILD_FLAG=$OPTARG
```

Comment:
only comment this out, why the entire if.

Reply:
Done